src/test/java/com/tradeshfit/HelloResourceTest.java

```diff
@@ -0,0 +1,92 @@
+package com.tradeshfit;
+
+import com.tradeshfit.model.dao.MessageDAO;
+import com.tradeshfit.model.dao.MessagesDAO;
+import com.tradeshift.HelloResource;
+import com.tradeshift.model.HelloResult;
+import com.tradeshift.model.RecentResult;
+import com.tradeshift.service.HelloService;
+import com.tradeshift.service.MessagesService;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.Assert;
+
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.List;
+
+import static org.mockito.Matchers.any;
+import static org.mockito.Matchers.eq;
+import static org.mockito.Mockito.*;
+
+public class HelloResourceTest {
+
+    private MessagesDAO messagesDAO;
+    private MessagesService messagesService;
+    private HelloService helloService;
+    private HelloResource resource;
+
+    @Before
+    public void SetUp(){
+        this.messagesDAO = mock(MessagesDAO.class);
+        this.messagesService = new MessagesService(messagesDAO);
+        this.helloService = mock(HelloService.class);
```

Comment:
These can also be done by adding the `@Mock` annotation to the fields, and then calling `MockitoAnnotations.initMocks(this)` in the setup method. Dealers choice.